### PR TITLE
feat: enabled BIGNUM

### PR DESCRIPTION
**Pull request by neeboo**
Enable BIGNUM support, related https://github.com/openwebf/webf/issues/12

modify CMakeLists.txt, adding CONFIG_BIGNUM and libbf.c
adding missing headers and types in bindings/qjs/qjs_engine_patch.h
modify dependencies in polyfill
modify assets.js to show it works
TODO: webf_websocket should change as well since it's also compiled by old qjsc

### CODE REVIEW

File: webf/macos/libquickjs.dylib [hunk 1]
```diff
@@ -1 +1 @@
-../../bridge/build/macos/lib/x86_64/libquickjs.dylib
\ No newline at end of file
+/Users/ghostcorn/dev/webf/webf/macos/../../bridge/build/macos/lib/x86_64/libquickjs.dylib
```
Comment (andycall) [inline webf/macos/libquickjs.dylib:1]: Don't commit this file

File: webf/macos/libwebf.dylib [hunk 2]
```diff
@@ -1 +1 @@
-../../bridge/build/macos/lib/x86_64/libwebf.dylib
\ No newline at end of file
+/Users/ghostcorn/dev/webf/webf/macos/../../bridge/build/macos/lib/x86_64/libwebf.dylib
```
Comment (andycall) [inline webf/macos/libwebf.dylib:1]: Same as previous

**Comment by andycall** [comment]
@neeboo

Webf-cli also deps on qjsc too. So a upgrade of webf-cli is needs. 
https://github.com/openwebf/cli

After the webf-cli had been upgraded, you can use it to regenerate new webf_websocket plugin. 


 https://github.com/openwebf/plugins/blob/feat/0.13.0/packages/webf_websocket/pubspec.yaml

**Comment by neeboo** [comment]
> @neeboo
> 
> Webf-cli also deps on qjsc too. So a upgrade of webf-cli is needed. https://github.com/openwebf/cli
> 
> After the webf-cli had been upgraded, you can use it to regenerate a new webf_websocket plugin.
> 
> https://github.com/openwebf/plugins/blob/feat/0.13.0/packages/webf_websocket/pubspec.yaml

I will revert this or and I had web-cli running locally with node-qjsc 0.3.0. Working fine

**Pull request closed without merging**